Frameworks/CoreGraphics/CGBitmapContext.mm

```diff
@@ -64,21 +68,25 @@ CGImageAlphaInfo CGBitmapContextGetAlphaInfo(CGContextRef context) {
 }*/
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 size_t CGBitmapContextGetBitsPerComponent(CGContextRef context) {
-    UNIMPLEMENTED();
-    return StubReturn();
+	  CGImageRef imageRef = context->Backing()->DestImage();
+	  size_t bitsPerComponent = CGImageGetBitsPerComponent(imageRef);
+      return bitsPerComponent;
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
+ @Notes UIView in Islandwood ignores alpha by default, which differs from iOS.
+ Expect returns of 24 bits on Islandwood where 32 bits is expected on iOS.
 */
 size_t CGBitmapContextGetBitsPerPixel(CGContextRef context) {
-    UNIMPLEMENTED();
-    return StubReturn();
+	CGImageRef imageRef = context->Backing()->DestImage();
+	size_t bytesPerPixel = imageRef->Backing()->BytesPerPixel();
+	size_t bitesPerCompoenent = CGBitmapContextGetBitsPerComponent(context);
+	return bytesPerPixel * bitesPerCompoenent;
```

Comment:
Should multiply by 8 here instead of bitsPerComponent - bytesPerPixel specifies bytes, not number of components